AUTHORS

```diff
@@ -0,0 +1,195 @@
+# Authors ordered by first contribution.
+
+## (generated with `git log --encoding=utf-8 --full-history --reverse "--format=format:%at;%an;%ae" | sort -t ';' -u -k 3,3 | sort | awk -F ';' '{ print $2 " <" $3 ">" }'`)
```

Comment:
unsure if it is necessary to have the file like this?

IIRC Node.js only has it for historical reasons but i may be incorrect.

The documents should probably just say how to get it out of git.